zerver/webhooks/jira/doc.md

```diff
@@ -1,20 +1,33 @@
-*If you are running JIRA version 5.2 or greater, or if you are
-using the hosted JIRA provided by Atlassian, you can use the built-in
-web-hook support to connect to Zulip — read on. For older,
-self-hosted JIRA installs, you can use our [JIRA Plugin](./jira-plugin).*
+Get Zulip notifications for your JIRA projects!
 
-{!create-stream.md!}
+!!! tip ""
+     If you are running JIRA version 5.2 or greater, or if you are
+     using the hosted JIRA provided by Atlassian, you can use the built-in
+     web-hook support to connect to Zulip — read on. For older,
+     self-hosted JIRA installs, you can use our [JIRA Plugin](./jira-plugin).
 
-{!create-bot-construct-url.md!}
+1. {!create-stream.md!}
 
-Next, in your JIRA administration control panel, go to the **Webhooks**
-page. If you are using the OnDemand hosted JIRA, follow the instructions
-[on the Atlassian wiki][1] for locating the Webhook UI.
+1. {!create-bot-construct-url-indented.md!}
 
-[1]: https://developer.atlassian.com/jiradev/jira-apis/webhooks#Webhooks-jiraadmin
+1. Go to your **Site administration** page, and click on **Jira** on the left.
```

Comment:
This should have them start with logging into Jira, right?

Reply:
Rishi recommended that we should leave out superfluous and obvious steps like "Log in to X" and assume that the user is logged in.

Reply:
We do want them to know that we're switching from Zulip to Jira though. We can do:
"Go to your Jira **Site administration** page. Click on **Jira** on the left. .."